Library/Formula/awsenv.rb

```diff
@@ -0,0 +1,66 @@
+require "language/go"
+
+class Awsenv < Formula
+  homepage "https://github.com/Luzifer/awsenv"
+  url "https://github.com/Luzifer/awsenv/archive/v0.3.1.tar.gz"
+  head "https://github.com/Luzifer/awsenv.git"
+  sha256 "106ff6acbe72bfc5e8a95a4cfbe49c5c34b4c682775e7072fe740cae8a7d3026"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/Sirupsen/logrus" do
+    url "https://github.com/Sirupsen/logrus",
```

Comment:
Used the suggested `homebrew-go-resources` tool and did not change in order to have it interchangeable without manual edit… Which variant do you prefer? The removed `:using => :git` or non-edited output from `homebrew-go-resources`? (Maybe also a change to that tooling to add `.git` to github.com URLs and remove the strategy?)